profiles/common/modules/features/multisite_notifications/multisite_notifications_core/includes/multisite_notifications_core.behat.inc

```diff
@@ -0,0 +1,83 @@
+<?php
+
+/**
+ * @file
+ * Contains \NexteuropaSubContext.
+ */
+
+use Drupal\DrupalDriverManager;
+use Drupal\DrupalExtension\Context\DrupalSubContextInterface;
+use Drupal\DrupalExtension\Context\RawDrupalContext;
+
+/**
+ * Behat step definitions for Nexteuropa modules.
+ */
+class NexteuropaSubContext extends RawDrupalContext implements DrupalSubContextInterface {
```

Comment:
I agree with @cyberwolf. Here we could use something like 'MultisiteNotificationSubContext' instead of 'NexteuropaSubContext' which is a bit misleading and can cause some issues in future.